src/SingleProject/Resizetizer/src/ResizetizeImages.cs

```diff
@@ -88,9 +88,7 @@ public override System.Threading.Tasks.Task ExecuteAsync()
 				}
 				catch (Exception ex)
 				{
-					LogWarning("MAUI0000", ex.ToString());
-
-					throw;
+					LogCodedError("MAUI0000", $"There was an exception processing the image '{img.Filename}': {ex}");
```

Comment:
Just to check I'm understanding: Because this is now logged as an error, that will cause the task from an MSBuild perspective to fail (and thus stop the build, or whatever), right?

Reply:
That is correct, yes.